server/src/test/java/org/opensearch/action/admin/indices/cache/clear/ClearIndicesCacheRequestTest.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.action.admin.indices.cache.clear;
+
+import junit.framework.TestCase;
+
+import static org.junit.Assert.assertThrows;
+
+public class ClearIndicesCacheRequestTest extends TestCase {
```

Comment:
Should probably extend OpenSearchTestCase (or some other opensearch test case class) rather than Junit's TestCase - not sure it'll get run by gradle otherwise?

Reply:
you are right, idk how i choose to extend from JUnit, it was probably copilot's code that I didn't change.